src/pages/MainPage/MainPage.style.jsx

```diff
@@ -0,0 +1,37 @@
+// MainPage.style.jsx
+import styled from 'styled-components';
+
+export const MainContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+export const WhiteBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  background: ${(props) => props.theme.color.white};
+  border-radius: 15px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  padding: 20px;
+  max-width: 100%;
+  box-sizing: border-box;
+`;
+
+export const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 25%;
+  max-width: 500px;
+  margin-left: 20px;
+`;
+
+export const AlbumUpdatesFeedContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 100%;
+  box-sizing: border-box;
+  overflow: hidden;
+  flex-grow: 1;
+`;
```

Comment:
제 노트북은 화면이 작아서 그런지 UI가 깨지네요 🥲
제가 한 번 수정해봤는데, 상혁님 화면에서도 잘 보이는지 확인해주세요!

<img width="1582" alt="image" src="https://github.com/user-attachments/assets/bbe97ee7-2c0d-4b16-b240-84bac265ca60">

```suggestion
// MainPage.style.jsx
import styled from 'styled-components';

export const MainContainer = styled.div`
  padding: 0 30px;
  display: flex;
  gap: 30px;
`;

export const WhiteBox = styled.div`
  flex: 1;
  display: flex;
  flex-direction: column;
  background: ${(props) => props.theme.color.white};
  border-radius: 15px;
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 20px;
  max-width: 100%;
  box-sizing: border-box;
`;

export const Content = styled.div`
  width: 400px;
  display: flex;
  flex-direction: column;
  gap: 30px;
`;

export const AlbumUpdatesFeedContainer = styled.div`
  display: flex;
  flex-direction: column;
  gap: 20px;
  width: 100%;
  box-sizing: border-box;
  overflow: hidden;
  flex-grow: 1;
`;
```